website/guides/05_rabbitmq.mdx

```diff
@@ -0,0 +1,473 @@
+    
+-   [Windows](https://www.rabbitmq.com/docs/install-windows)
+    
+-   [MacOS](https://www.rabbitmq.com/docs/install-homebrew)
+    
+
+### RabbitMQ Cluster Kubernetes Operator ☸️
+
+We will focus on this installation method since it’s the recommended method for working with RabbitMQ in Kubernetes environments.
+
+## Install the RabbitMQ Cluster Kubernetes Operator
+
+The RabbitMQ Cluster Kubernetes Operator automates the provisioning, management, and operations of RabbitMQ clusters running on Kubernetes.
+
+## Installation
+
+Installing the RabbitMQ cluster operator can easily be achieved with the Glasskube package manager.
+
+### Install Glasskube
+
+If you already installed `glasskube` you can skip this step.
+If not, `glasskube` can easily be installed the way you usually install packages for your operating system.
+
+<Install/>
+
+### Install the RabbitMQ cluster operator
+
+<Tabs groupId="interface">
+  <TabItem value="gui" label="GUI 🖥️">
+    Start the UI via the command line:
+
+    ```shell
+    glasskube serve
+    ```
+
+    Install RabbitMQ cluster operator via the Glasskube UI.
+  </TabItem>
+  <TabItem value="cli" label="CLI 🧑‍💻">
+    A package can be installed with a simple command.
+
+    ```shell
+    glasskube install rabbitmq-operator
+    ```
+
+    The process will wait until the package got successfully installed.
+  </TabItem>
+</Tabs>
+
+You will need access to a Kubernetes cluster running `Kubernetes 1.19` or later (You can easily create a local cluster by using [Minikube](https://minikube.sigs.k8s.io/docs/)) `kubectl` isn’t strictly speaking a dependency for installing packages via Glasskube, but it is the recommended way to interact with the cluster. Therefore, it is highly recommended. Installation instructions are available for macOS, Linux, and Windows.
+
+> ℹ️ Glasskube will automatically install the cluster operator in a newly created namespace called `rabbitmq-system`
+
+## Confirm Service Availability[](https://www.rabbitmq.com/kubernetes/operator/using-operator#service-availability "Direct link to Confirm Service Availability")
+
+Before configuring your app to use RabbitMQ Cluster Kubernetes Operator, ensure that `RabbitmqCluster` Custom Resource is deployed to your Kubernetes cluster and is available.
+
+To confirm this availability, run
+
+```
+kubectl get customresourcedefinitions.apiextensions.k8s.io
+```
+
+## Create a RabbitMQ Instance[](https://www.rabbitmq.com/kubernetes/operator/using-operator#create "Direct link to Create a RabbitMQ Instance")
+
+The RabbitMQ Cluster Kubernetes Operator creates the necessary resources, such as `Services`, `StatefulSets`, `Secrets` and `ConfigMaps` in the same namespace in which the `RabbitmqCluster` was defined.
+
+First, create a YAML file to define a `RabbitmqCluster` resource named definition.yaml.
+
+> ℹ️ **Note:** The YAML file can have any name, but the steps that follow assume it is called `definition`.
```

Comment:
I would say "... it is called `definition.yaml`" then, since you are referring to it like that below too. Plus the name of the cluster is also `definition` which could lead to confusion.